martin/src/srv/server.rs

```diff
@@ -98,16 +100,14 @@ pub fn router(cfg: &mut web::ServiceConfig) {
     cfg.service(crate::srv::fonts::get_font);
 }
 
-/// Create a new initialized Actix `App` instance together with the listening address.
-pub fn new_server(config: SrvConfig, state: ServerState) -> MartinResult<(Server, String)> {
+/// Create a future for an Actix web server together with the listening address.
+pub fn new_server(
+    config: SrvConfig,
+    state: ServerState,
+) -> MartinResult<(LocalBoxFuture<'static, MartinResult<()>>, String)> {
```

Comment:
same thing as above - any way to simplify the result of this, or at least lots of comments?